service/src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java

```diff
@@ -159,26 +166,17 @@ public class CloudSyncRoleMapping {
           "dataproc.workflowTemplates.delete",
           "dataproc.workflowTemplates.instantiate",
           "dataproc.workflowTemplates.instantiateInline",
-          "dataproc.operations.get",
-          "dataproc.operations.list",
           "dataproc.operations.cancel",
           "dataproc.operations.delete",
           "dataproc.tasks.lease",
           "dataproc.tasks.listInvalidatedLeases",
           "dataproc.tasks.reportStatus",
-          "dataproc.agents.get",
-          "dataproc.agents.list",
           "dataproc.agents.create",
           "dataproc.agents.update",
           "dataproc.agents.delete",
-          "compute.machineTypes.get",
           "compute.networks.get",
```

Comment:
Do you also need to move `compute.networks.get`, `compute.networks.list`, and/or `compute.projects.get`?